static/app/views/explore/logs/logsTable.tsx

```diff
@@ -0,0 +1,59 @@
+import GridEditable from 'sentry/components/gridEditable';
+import TimeSince from 'sentry/components/timeSince';
+import type {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useOurlogs} from 'sentry/views/insights/common/queries/useDiscover';
+import type {OurlogsFields} from 'sentry/views/insights/types';
+
+export type LogsTableProps = {
+  search: MutableSearch;
+};
+
+export function LogsTable(props: LogsTableProps) {
+  const {data, error, isPending} = useOurlogs(
+    {
+      limit: 30,
```

Comment:
I assume this 30 was arbitrarily chosen? 30 feels awfully little in the context of logs